tests/test_json.ml

```diff
@@ -0,0 +1,60 @@
+let read_file fname =
+  (* the tests are run from the _build/default/tests directory *)
+  Fpath.(v ".." / ".." / ".." / "tests" / fname)
+  |> Fpath.to_string |> Yojson.Basic.from_file
+
+let dune_release_testing_json = read_file "dune-release-testing.json"
+
+let upload_response_json = read_file "upload-response.json"
+
+let create_release_response_json = read_file "create-release-response.json"
+
+let pull_request_response_json = read_file "pull-request-response.json"
+
+let archive_upload_url () =
+  let check json expected =
+    match Dune_release.Json.Upload_response.browser_download_url json with
+    | Ok actual -> Alcotest.(check string) __LOC__ actual expected
+    | Error (`Msg msg) -> Alcotest.fail msg
+  in
+  check dune_release_testing_json
```

Comment:
Could you please rewrite those tests so they to stick to the testing style used in the other files?

That is, tests for a given function are a list of test cases rather than a sequence of `check` calls. I understand it's a bit annoying but it's better for both consistency and better test failure reporting from `Alcotest`. A sequence of checks will stop at the first failing test case where a list of test case will give you the full report no matter what.

Reply:
done